ratis-docs/content/cli.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Ratis-shell
+menu: main
+weight: -10
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Ratis-shell 
+Ratis's commad line interface provides operations to manage the ratis server. You can invoke the following command line 
+utility to get all the subcommands:
+
+```shell
+$ cd $PATH_TO_RATIS/ratis-assembly/target/apache-ratis-$version-SNAPSHOT
+$ ./bin/ratis sh
```

Comment:
Let's simply use
```
$ ratis sh
```
We may assume `ratis` is already in the PATH.

Reply:
done